.github/workflows/ci_dockers.yml

```diff
@@ -122,11 +122,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # the config used in 'dockers/ipu-ci-runner/Dockerfile'
-        python_version: ["3.9"]  # latest
-        # TODO: upgrade - PopTorch 2.2 uses torch 1.9, see:
```

Comment:
@Borda IPU CI installs PopTorch 2.1 which requires PyTorch 1.7

So for us to drop PyTorch 1.7, we need to use PyTorch 1.9 in IPU CI as Kush already did.

But where do we configure IPU CI to install PopTorch 2.2?